### PR TITLE
fix: remove extra backward slash #780

### LINKED ISSUE #780: [DOCS] Remove extra backward slash
### Remove extra backward slash

* Line Permanent [Link](https://github.com/EddieHubCommunity/LinkFree/blob/be46fba3fd551fd23cc089d7b6ea376b2e29e64f/README.md?plain=1#L131) 

Before Changes:

`For example: http://linkfree.eddiehub.org/eddiejaoude\`

After Changes:

`For example: http://linkfree.eddiehub.org/eddiejaoude`

### Screenshots

Screenshot from 2021-12-08 08-21-44

### Additional information

N/A

### PR DESCRIPTION
**Pull request by VaibhavUpreti**
[README.md](https://github.com/EddieHubCommunity/LinkFree/files/7673340/README.md)
[README.md](https://github.com/EddieHubCommunity/LinkFree/files/7673342/README.md)

Closes #780 

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/782"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM 🚀
Thanks for the contribution.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please make the changes in `taravis`' profile in another branch and another PR. Let us know if you need any help. 🙂

**Comment by VaibhavUpreti** [comment]
> Please make the changes in `taravis`' profile in another branch and another PR. Let us know if you need any help. 🙂
I might need a little help could you guide me about that.

**Comment by VaibhavUpreti** [comment]
I am not able to understand how to make a new pull request.

**Comment by Panquesito7** [comment]
> I am not able to understand how to make a new pull request.

Please follow the steps mentioned in #779. 🙂

**Comment by VaibhavUpreti** [comment]
I just did the steps in #779. Is it correct now?

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Please revert taravis.json

File: public/data/taravis.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "Anu Wat",
+  "type": "personal",
```
Comment (schmelto) [inline public/data/taravis.json:3]:
```suggestion
```

**Review by schmelto — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]

**Review by Vyvy-vi — APPROVED** [review]
looks alright!

**Pull request merged**